### PR TITLE
Added Validation

### PR DESCRIPTION
**Pull request by anuj-kumary**
Feat: Added Note Validation

Preview:- https://deploy-preview-14--eagle-notes.netlify.app/

### CODE REVIEW

**Review by vinc3nati — APPROVED** [review]
Good to see you are handling errors

**Review by Rohanmond — APPROVED** [review]
code looks good to merge

File: src/components/Main/Main.jsx [hunk 1]
```diff
@@ -33,35 +34,45 @@ export const Main = () => {
     setExpand(false);
   };
 
+  const showErrorMsg = () => {
+    setError(true);
+  };
+  console.log(error);
```
Comment (varnita619) [inline src/components/Main/Main.jsx]: remove this console if not necessary

**Review by varnita619 — APPROVED** [review]
Code is clean
UI looks nice
Folder structure is good with proper naming convensions
Ready to merge

File: src/components/Main/Main.jsx (hunk 1 above)
Comment (anuj-kumary) [reply src/components/Main/Main.jsx]: Okay

**Review by PeeyushPrashant — APPROVED** [review]

**Pull request merged**